en/activity-standard/iati-activities/iati-activity/transaction/aid-type.rst

```diff
@@ -4,10 +4,17 @@ Example usage of ``aid-type`` of a ``transaction`` in an ``iati-activity``.
 
 | The ``@code`` attribute declares a valid code (*A02*) from the *AidType* codelist.
 
-| Note: The ``aid-type`` element can override the ``default-aid-type`` value set in ``iati-activity``: 
+| Note: The ``aid-type`` element can override the ``default-aid-type`` value set in ``iati-activity``:
 
 .. literalinclude:: ../../../activity-standard-example-annotated.xml
 	:language: xml
 	:start-after: <!--transaction starts-->
 	:end-before: <!--transaction ends-->
 	:emphasize-lines: 21
```

Comment:
Includes an :emphasize-lines, but would suggest #413 is re-run in order to remove these en masse after this and other PRs are merged into v2.03 dev.

Reply:
Fixed in 8fb4761.